appservice/src/intTest/groovy/nu/yona/server/BasicBuddyTest.groovy

```diff
@@ -80,6 +80,7 @@ class BasicBuddyTest extends AbstractAppServiceIntegrationTest
 		def buddyConnectRequestMessages = response.responseData._embedded.messages.findAll{ it."@type" == "BuddyConnectRequestMessage"}
 		buddyConnectRequestMessages.size() == 1
 		buddyConnectRequestMessages[0].nickname == richard.nickname
+		assertDateTimeInLimits(buddyConnectRequestMessages[0].creationTime)
```

Comment:
I think `assertDateEquals(buddyConnectRequestMessages[0].creationTime, new Date(), 10)` is more understandable